src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -13,17 +13,15 @@ import { apiRequest } from '../../../platform/utilities/api';
 import { genderLabels } from '../../../platform/static-data/labels';
 import { getDiagnosticCodeName } from './reference-helpers';
 
-const siblings = ['treatments', 'privateRecordReleases', 'privateRecords', 'additionalDocuments'];
-
 import { PREFILL_STATUSES } from '../../common/schemaform/save-in-progress/actions';
 import { DateWidget } from '../../common/schemaform/review/widgets';
 
-const vaForm4142URL = 'https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf';
-
-export const USA = 'USA';
-export const MILITARY_STATES = ['AA', 'AE', 'AP'];
-export const MILITARY_CITIES = ['APO', 'DPO', 'FPO'];
+import {
+  USA,
+  VA_FORM4142_URL
+} from './constants';
 
+const siblings = ['treatments', 'privateRecordReleases', 'privateRecords', 'additionalDocuments'];
```

Comment:
Should we use something like "uploadItems" or "documentTypes" here instead?

Reply:
No idea 😛 This is actually existing code that just moved down a few lines. I have to admit this is kind of a messy diff 😞

Reply:
Thanks for pointing that out, I wonder whether we should keep in mind the work that @goldenmeanie is doing to plan the document upload UI, and the types of variable names we are likely to want to use for that. Right now, the name used in the common fileUploadUI is `documentId` (for document type), so maybe avoiding documentXYZ, in favor or recordType or recordXYZ might be helpful to distinguish the meaning here? Not sure, naming is hard.

Reply:
Yeah, I'd probably vote to split out renaming this into whatever PR / issue we end up making for the rest of the document upload work.